models/release.go

```diff
@@ -397,13 +397,15 @@ func UpdateRelease(doer *User, gitRepo *git.Repository, rel *Release, attachment
 		return err
 	}
 
+	if err = addReleaseAttachments(rel.ID, attachmentUUIDs); err != nil {
+		log.Error("addReleaseAttachments: %v", err)
+	}
+
 	err = rel.loadAttributes(x)
 	if err != nil {
```

Comment:
Not strictly part of your backport, I know, but this should be simplified as:
```
if err = rel.LoadAttributes(); err != nil {
```

Reply:
@guillep2k this should be done in other PR to master